src/algebra/lie_algebra.lean

```diff
@@ -800,8 +777,7 @@ local attribute [instance] matrix.lie_ring
 
 /-- An important class of Lie algebras are those arising from the associative algebra structure on
 square matrices over a commutative ring. -/
-def matrix.lie_algebra : lie_algebra R (matrix n n R) :=
-lie_algebra.of_associative_algebra (matrix n n R)
+def matrix.lie_algebra : lie_algebra R (matrix n n R) := by apply_instance
```

Comment:
Similarly to with the endomorphisms, do we still need `matrix.lie_ring` and `matrix.lie_algebra`?

Reply:
In the case of matrices it could make sense to leave it just to say "Hey look, in particular I proved this". I did it for example with real numbers in the section of Lie groups. However, I agree it is better to remove it so unless you comment back I'll remove it